lib/trento/application/read_models/host_read_model.ex

```diff
@@ -32,6 +32,10 @@ defmodule Trento.HostReadModel do
       foreign_key: :host_id,
       preload_order: [desc: :identifier]
 
+    has_one :heartbeat_timestamp, Trento.Heartbeat,
```

Comment:
I will replace :heartbeat_timestamp with :hearbeat as association name, when defining association, it's better to stick with the entity name, in this case hearbeat, we know that's is only one because is a 1:1 association, and it does not appear that we perform some select or manipulation on it, so it' s not a timestamp, it's an heartbeat.